packages/flutter_tools/templates/app/pubspec.yaml.tmpl

```diff
@@ -44,6 +49,12 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
 
+  # The following imports a recommended set of lints designed to encourage good
+  # coding practises. Lints are configured and activated in the
+  # analysis_options.yaml file located at the root of your package.
+  lints:
+    path: /Users/goderbauer/dev/lints # TODO(goderbauer): Update this before submitting.
```

Comment:
isn't this redundant with the analysis options? i'm not sure i understand how they relate. if we really need both we should definitely explain the difference in these files.

Reply:
Reworded the paragraph above. This line just makes the set of recommended lints available. The import in the `analysis_options.yaml` then activates that set.